ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -117,6 +117,10 @@ message CreatedEvent {
   // after the introduction of explicit disclosure.
   // Optional
   ContractMetadata metadata = 10;
+
+  // Maintainers of the contract key, if defined.
+  // Optional
```

Comment:
If this is backed onto the key maintainers I added it may be best to document that this field will only be populated for events created after version 2.7.0.

Reply:
Good point. Added